dev/acf-export.json

```diff
@@ -242,43 +242,141 @@
         "description": ""
     },
     {
-        "key": "group_5e627619cf1b7",
+        "key": "group_5e6a6df06c71e",
         "title": "Research Cluster Fields",
         "fields": [
             {
-                "key": "field_5e68f030e0d6e",
-                "label": "Cluster Leads",
-                "name": "cluster_leads",
+                "key": "field_5e6b94250d197",
+                "label": "News Section",
+                "name": "",
+                "type": "tab",
+                "instructions": "",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "placement": "top",
+                "endpoint": 0
+            },
+            {
+                "key": "field_5e6b94390d198",
+                "label": "News Stories",
+                "name": "cluster_news_stories",
                 "type": "repeater",
-                "instructions": "List the lead faculty members of this cluster.",
+                "instructions": "Select the news stories to display for this cluster.",
                 "required": 0,
                 "conditional_logic": 0,
                 "wrapper": {
                     "width": "",
                     "class": "",
                     "id": ""
                 },
-                "collapsed": "field_5e68f05de0d6f",
-                "min": 1,
+                "collapsed": "field_5e6b945d0d199",
+                "min": 0,
                 "max": 0,
                 "layout": "row",
                 "button_label": "",
                 "sub_fields": [
                     {
-                        "key": "field_5e68f05de0d6f",
-                        "label": "Cluster Lead",
-                        "name": "cluster_lead",
-                        "type": "post_object",
-                        "instructions": "Select the faculty cluster lead.",
+                        "key": "field_5e6b945d0d199",
+                        "label": "Story Source",
+                        "name": "story_source",
+                        "type": "radio",
+                        "instructions": "Select the source of the news story",
```

Comment:
Small beans, but, most of the instruction lines have periods at the end. Can we add them to all of them for consistency?

Reply:
Small beans corrected!